ogarniacz_mp3.py

```diff
@@ -86,10 +81,16 @@ def processing_to_verification(processing_dir_path, verification_dir_path):
                             # e) znormalizować tagi
                             # f) pozbyć się zbędnych tagów
                             # 2. przeniesienie ich do verification
-                            try:
-                                shutil.move(file_path, destination_pth)
-                            except FileNotFoundError:
-                                continue #jeśli pliku nie ma, to nie można go przenieść
+                    for dir_path, sub_dirs, files in os.walk(processing_dir_path, topdown=False):
+                        dir_name = os.path.basename(dir_path)
+                        print(dir_name)
+                        current_pth = os.path.join(processing_dir_path, dir_name)
+                        destination_pth = os.path.join(verification_dir_path, dir_name)
+                        os.makedirs(destination_pth, exist_ok=True)
+                        try:
+                            shutil.move(current_pth, destination_pth)
+                        except FileNotFoundError:
+                            continue #jeśli pliku nie ma, to nie można go przenieść
```

Comment:
Robisz przenoszenie katalogów wewnątrz pętli do usuwania duplikatów. Dlatego efekty są... właściwie to nawet trudne do ogarnięcia.

Wszystko od `# b) przekonwertować nie mp3 na mp3` powinno być wyjęte z jakiejkolwiek wcześniejszej pętli. 
Każdy `# podpunkt` powinien być wcięty na ten sam poziom co  `# a) pozbyć się duplikatów`.

Reply:
Rozumiem dlaczego przenoszenie katalogów ma być poza poprzednimi pętlami, ale nie rozumiem, dlaczego konwertowanie też.

Reply:
Bo dopóki nie usuniesz duplikatów, nie masz pewności, że jest sens coś konwertować.

Przepuśćmy, że masz taką samą sytuację jak rozważaliśmy do usuwania duplikatów - masz dwa pliki z tym samym utworem - jeden w formacie mp3, drugi w formacie flac. Nie ma sensu konwertować flac na mp3 - już mamy taki plik mp3.

Dodatkowo w tamtej pętli nie ma dobrego miejsca, żeby to zrobić. Tak jak masz to napisane teraz, `if ext != "mp3":` jest w ogóle sprawdzane tylko, jeśli został znaleziony duplikat. A co jeśli mam same pliki flac i nie ma duplikatów?